lib/bmg/writer/csv.rb

```diff
@@ -0,0 +1,31 @@
+module Bmg
+  module Writer
+    class Csv
+      include Writer
+
+      DEFAULT_OPTIONS = {
+      }
+
+      def initialize(options)
+        @options = DEFAULT_OPTIONS.merge(options)
+      end
+      attr_reader :options
+
+      def call(r, output = nil)
```

Comment:
Is `r` used consistently as a variable name for relations? If not, IMO it's unnecessarily terse, I would prefer `relation` or at least `rel`.